en/docs/manage/api-lifecycle.md

```diff
@@ -1,4 +1,4 @@
 # API Lifecycle
 
 !!! note
-    This section will disucss about the API Lifecycle states and its usage in Choreo. 
+    This section will describe the states of the API Lifecycle and the usage of the API lifecycle in Choreo. 
```

Comment:
Why are we using two capitalisations for Lifecycle?

Reply:
How about "This section will describe the API lifecycle, its states, and its usage in Choreo."